maps4fs/generator/dtm/lithuania.py

```diff
@@ -0,0 +1,71 @@
+"""This module contains provider of Lithuania data."""
+
+from typing import List
+import requests
+import os
+import tempfile
+import numpy as np
+from maps4fs.generator.dtm.dtm import DTMProvider
+
+class LithuaniaProvider(DTMProvider):
+    """Provider of Lithuania data."""
+
+    _code = "lithuania"
+    _name = "Lithuania"
+    _region = "LT"
+    _icon = "🇱🇹"
+    _resolution = 1.0
+    _author = "[Tox3](https://github.com/Tox3)"
+    _is_community = True
+    _is_base = False
+    _extents = (56.4501789128452, 53.8901567283941, 26.8198345671209, 20.9312456789123)
+    _max_tile_size = 4096
+
+    def get_tile_path(self, filename: str) -> str:
+        temp_dir = tempfile.mkdtemp()
+        return os.path.join(temp_dir, filename)
```

Comment:
The base class already creates and exposes a `_tile_directory` path, but if you use the `download_tif_files` method (see my comment below) then that will take care of that for you.